src/kudu/util/oid_generator-test.cc

```diff
@@ -26,9 +26,12 @@
 
 using std::string;
 
+DECLARE_bool(cononicalize_uuid);
+
 namespace kudu {
 
 TEST(ObjectIdGeneratorTest, TestCanoicalizeUuid) {
+  FLAGS_cononicalize_uuid = true;
```

Comment:
Before you change the flag, add:

    google::FlagSaver saver;

See hybrid_clock-test.cc for an example